src/service/data_products/taxa_count.py

```diff
@@ -250,10 +250,13 @@ async def get_taxa_counts(
     if rank not in ranks.data:
         raise errors.IllegalParameterError(f"Invalid rank: {rank}")
     q = await _query(store, collection_id, load_ver, rank)
```

Comment:
Can we change `q` to something else? I keep reading it as query rather than what it is, the result

Reply:
👍

Reply:
:+1: